lib/cronitor/config.rb

```diff
@@ -13,4 +13,9 @@ def configure(&block)
   self.config = ENV['CRONITOR_CONFIG']
   self.logger = Logger.new(STDOUT)
   self.logger.level = Logger::INFO
+  self._headers = {
+    'Content-Type': 'application/json',
+    'User-Agent': 'cronitor-ruby',
+    'Cronitor-Version': Cronitor.api_version
+  }
```

Comment:
👏🏻

Reply:
Hmm; if `Cronitor#configure` is a method on the main module, is there a reason it's broken out into its own file like this?

As in, I would expect this to be defined in `lib/cronitor.rb`, since it’s not in a subclass/submodule of the top-level on.

Reply:
@jeffbyrnes think it's worth keeping the #configure method, or does allowing 

```
Cronitor.api_key = XXX
```

make that no longer necessary?